web/src/app/app.module.ts

```diff
@@ -48,6 +48,7 @@ import { SocialCardsComponent } from './components/social-cards/social-cards.com
     UserService,
     ErrorService,
     AuthGuardService,
+    HttpClientModule
```

Comment:
I don't think a module is needed here?

Reply:
Okay 👍

Reply:
I think it needs to be under the `imports` but not the `providers`. But if it still works then it is fine I guess